ts/features/itwallet/presentation/screens/ItwPresentationEidVerificationExpiredScreen.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useMemo } from "react";
+import { BodyProps } from "@pagopa/io-app-design-system";
+import { OperationResultScreenContent } from "../../../../components/screens/OperationResultScreenContent";
+import I18n from "../../../../i18n";
+import { useIONavigation } from "../../../../navigation/params/AppParamsList";
+import { useHeaderSecondLevel } from "../../../../hooks/useHeaderSecondLevel";
+import { ITW_ROUTES } from "../../navigation/routes";
+
+export const ItwPresentationEidVerificationExpiredScreen = () => {
+  const navigation = useIONavigation();
+
+  useHeaderSecondLevel({
+    title: "",
+    headerShown: false
+  });
```

Comment:
No need to use the hook. We can remove the header in the screen options within the `ItwStackNavigator` component. What do you think?